blog/content/post/2021-02-21-amd-milan-osf.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Open Source Firmware on AMD Milan server processors
+abstract: 'There were times where AMD was actively supporting open source
+           firmware ecosystem by providing silicon initialization code. With
+           a few years break AMD is now trying to support open source firmware
+           again with the mobile platform like Chromebooks. However, the recent
+           achievements have made everybody think that open source firmware is
+           also possible on severs.'
+cover: /covers/amd-epyc.jpg
+author: michal.zygowski
+layout: post
+published: true
+date: 2021-02-21
+archives: "2021"
+
+tags:
+  - firmware
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+Do you remember times when AMD was actively supporting open source firmware?
+Back in the 2008 AMD was delivering the AGESA silicon initialization code and
+was contributing it to coreboot. Up to 2014 AMD was release the AGESA code in
+public. These were the golden years of open source firmware for AMD processors
+and many platforms emerged in coreboot. You may know those platform under
+various names like PC Engines ALIX and APU1. They benefit still from fully open
+source implementation. However since 2014 AMD started to deliver the AGESA in
+binary form only. And that situation is maintained till today with some
+exceptions (after AMD Steppe Eagle there were no AGESA binary releases
+anymore). Google encouraged the AMD to work on FSP compatible interface for
+AGESA for their Chromebooks and today we have a AMD Picasso support in
+coreboot. If you want to know more about Open Source Firmware on AMD history,
+please refer to our FOSDEM presentations:
+
+* https://video.fosdem.org/2020/K.4.401/coreboot_amd.mp4
+* https://video.fosdem.org/2021/D.firmware/firmware_osfsoap2.mp4
```

Comment:
@miczyg1 let's hide links behind full presentatation names or link to specific office section.

Reply:
Done